src_c/scale.h

```diff
@@ -29,10 +29,11 @@
 #if !defined(SCALE_HEADER)
 #define SCALE_HEADER
 
-#if (defined(__GNUC__) &&                                      \
-     ((defined(__x86_64__) && !defined(_NO_MMX_FOR_X86_64)) || \
-      defined(__i386__))) ||                                   \
-    (defined(MS_WIN32) && !(defined(_M_X64) && defined(_NO_MMX_FOR_X86_64)))
+#if (defined(__GNUC__) &&                                             \
+     ((defined(__x86_64__) && !defined(_NO_MMX_FOR_X86_64)) ||        \
+      defined(__i386__))) ||                                          \
+    (defined(MS_WIN32) && !((defined(_M_X64) || defined(_M_ARM64)) && \
+                            defined(_NO_MMX_FOR_X86_64)))
```

Comment:
Sorry for the delay in handling this @cgohlke 

I'm having a hard time understanding what this guard is doing.

With some bubble pushing, the last bit of logic becomes
`(!defined(_M_X64) && !defined(_M_ARM64)) || !defined(_NO_MMX_FOR_X86_64)`
And if that is True and its on Windows, it will include the mmx support.

So if _NO_MMX_FOR_X86_64 is undefined, it will always be True and therefore the MMX stuff will always be included?

Reply:
The patch doesn't change the logic much, just added Windows on ARM64 to the list of platforms for which, when `_NO_MMX_FOR_X86_64` is defined, MMX will not be used. So yes, `-D_NO_MMX_FOR_X86_64` will need to be included in `Setup`, but at least with this patch the definition does what is expected. Without this patch, MMX will always be included and of course fail on Windows on ARM64.

Reply:
Okay, that makes sense. Thanks.